client/src/components/StatisticsTab.jsx

```diff
@@ -33,24 +38,96 @@ export default function StatisticsTab({ networkStatistics }) {
         </TabList>
         <TabPanel>
           <h4 className="stats-title">Graph stats</h4>
-          <div className="graph-statistics-container">
-            <div>
-              <ul className="right-aligned-list">
-                <li># of nodes:</li>
-                <li># of edges:</li>
-                <li># of paths from source:</li>
-                <li>Average node degree:</li>
-              </ul>
+          <div className="graph-stats-container">
+            <div className="img-text-container1">
```

Comment:
the classnames img-text-container1, img-text-container2, img-text-container3 have the same CSS attributes and do the same thing. you can condense this into just one css classname, say image-text-container, and applying to all three.